app/src/main/kotlin/com/github/gotify/init/InitializationActivity.kt

```diff
@@ -1,12 +1,17 @@
 package com.github.gotify.init
 
```

Comment:
Sorry for taking so long to review this (:. I'm having trouble getting it to work on Android 12/API 31 and Android 12l/API 32. With the current state of master it works fine, but with these changes it crashes after the login, or shows a blank page:
[android12.webm](https://github.com/gotify/android/assets/14895212/11bd004a-8e44-4a02-b0e3-b159c553f9f1)

API 30,33,34 works fine. There is no exception logged, after pressing login & creating the client.

Reply:
Interesting, sorry for the problems.
I will take a look as soon as possible.

Reply:
I think I've found my error.  
Works now on my emulators on SDK 25, 31, 34.

Reply:
Nice, I've tested it with SDK 23,30,31,32,33 & 34 and it works fine. Thanks for debugging and fixing this (:. The dependency problem was also fixed somehow, so I'll don't change the logging dependency yet.